### PR TITLE
[3.0][docs] Add documents for MySql pipeline connector

### PR DESCRIPTION
**Pull request by Jiabao-Sun**
Add documents for MySql pipeline connector

### CODE REVIEW

File: docs/content/pipelines/mysql-pipeline(ZH).md [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+
+```yaml
+source:
+   type: mysql
+   name: MySQL Source
+   hostname: 127.0.0.1
+   port: 3306
+   username: admin
+   password: pass
+   table: adb.\.*, bdb.user_table_[0-9]+, [app|web].order_\\.*
```
Comment (ruanhang1993) [inline docs/content/pipelines/mysql-pipeline(ZH).md:20]: Why is `adb.\.*` using `\` and `[app|web].order_\\.*` using `\\`?

File: docs/content/pipelines/mysql-pipeline(ZH).md [hunk 2]
```diff
@@ -0,0 +1,547 @@
...
+      <td>Duration</td>
+      <td>用于跟踪最新可用 binlog 偏移的发送心跳事件的间隔。</td>
+    </tr>
+    <tr>
+      <td>debezium.*</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>将 Debezium 的属性传递给 Debezium 嵌入式引擎，该引擎用于从 MySQL 服务器捕获数据更改。
+          For example: <code>'debezium.snapshot.mode' = 'never'</code>.
```
Comment (ruanhang1993) [inline docs/content/pipelines/mysql-pipeline(ZH).md:202]: 例如：

File: docs/content/pipelines/mysql-pipeline(ZH).md [hunk 3]
```diff
@@ -0,0 +1,547 @@
...
+      <td>TIMESTAMP [(p)]
+      </td>
+      <td>TIMESTAMP [(p)]
+      </td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>DATETIME [(p)]
+      </td>
+      <td>TIMESTAMP_LTZ [(p)]
```
Comment (ruanhang1993) [inline docs/content/pipelines/mysql-pipeline(ZH).md:372]: DATETIME im mapped to TIMESTAMP and TIMESTAMP is mapped to TIMESTAMP_LTZ

File: docs/content/pipelines/mysql-pipeline.md [hunk 4]
```diff
@@ -0,0 +1,554 @@
...
+
+```yaml
+source:
+   type: mysql
+   name: MySQL Source
+   hostname: 127.0.0.1
+   port: 3306
+   username: admin
+   password: pass
+   table: adb.\.*, bdb.user_table_[0-9]+, [app|web].order_\.*
```
Comment (GOODBOY008) [inline docs/content/pipelines/mysql-pipeline.md:20]:
```suggestion
   tables: adb.\.*, bdb.user_table_[0-9]+, [app|web].order_\.*
```

File: docs/content/pipelines/mysql-pipeline(ZH).md [hunk 5]
```diff
@@ -0,0 +1,547 @@
...
+        BLOB<br>
+        MEDIUMBLOB<br>
+        LONGBLOB<br>
+      </td>
+      <td>BYTES</td>
+      <td>目前，对于 MySQL 中的 BLOB 数据类型，仅支持长度不大于 2147483647（2**31-1）的 blob。 </td>
+    </tr>
+    <tr>
+      <td>
+        YEAR
```
Comment (ruanhang1993) [inline docs/content/pipelines/mysql-pipeline(ZH).md:433]: `YEAR` is already described in the previous part.

File: docs/content/pipelines/mysql-pipeline.md [hunk 6]
```diff
@@ -0,0 +1,554 @@
+# MySQL CDC Pipeline Connector
+
+The MySQL CDC connector allows for reading snapshot data and incremental data from MySQL database and provides end-to-end full-database data synchronization capabilities.
```
Comment (ruanhang1993) [inline docs/content/pipelines/mysql-pipeline.md:3]: MySQL CDC Pipeline Connector

**Review by GOODBOY008 — APPROVED** [review]
LGTM @ruanhang1993 PTAL

**Review by ruanhang1993 — APPROVED** [review]
LGTM.

**Pull request merged**